packages/docusaurus-theme-classic/src/theme/DocSidebar/index.tsx

```diff
@@ -46,6 +46,8 @@ function DocSidebarItemCategory({
   onItemClick,
   collapsible,
   activePath,
+  collapseItems,
+  autoCollapsed,
```

Comment:
i would keep this property name consistent with the setting name,
Ex. `autoCollapse` or `autoCollapseCategory`

Reply:
agree :)

Reply:
Renamed these properties.